local-spa/palette.html

```
@@ -0,0 +1,490 @@
+  <header>
+    <h1>SKY UX Palette - Gray & Slate</h1>
+    <nav class="view-nav">
+      <a href="./index.html#/surfaces">Surface Demo</a>
+      <a href="./index.html#/palette">Palette (Embedded)</a>
+      <a href="./palette.html" class="is-active">Palette (Detailed)</a>
+      <a href="./index.html?local-preview=layers">Carbon Layering</a>
+    </nav>
+  </header>
+
+  <main>
+    <div class="container">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div> ], start tag match failed [ <div class="container"> ] on line 215.

(tag-pair)